lib/src/util/validationUtil.ts

```diff
@@ -42,3 +43,7 @@ export function getFirstValidationError(validationErrors: ValidationErrors | und
     return validationErrors;
   }
 };
+
+export function combineValidationErrors(...validationErrors: (ValidationErrors | undefined)[]): ValidationErrors {
```

Comment:
Realized after posting the PR that this needs docs. I'll add them before this is merged.  TODO

Reply:
> Realized after posting the PR that this needs docs. I'll add them before this is merged. TODO

d44ef6da